internal/eval/eval.go

```diff
@@ -660,6 +656,79 @@ func evalJSONGET(args []string, store *dstore.Store) []byte {
 	return clientio.Encode(string(resultBytes), false)
 }
 
+// evalJSONTOGGLE toggles a boolean value stored at the specified key and path.
+// args must contain at least the key and path (where the boolean is located).
+// If the key does not exist or is expired, it returns response.RespNIL.
+// If the field at the specified path is not a boolean, it returns an encoded error response.
+// If the boolean is `true`, it toggles to `false` (returns :0), and if `false`, it toggles to `true` (returns :1).
+// Returns an encoded error response if the incorrect number of arguments is provided.
+func evalJSONTOGGLE(args []string, store *dstore.Store) []byte {
+    if len(args) < 2 {
+        return diceerrors.NewErrArity("JSON.TOGGLE")
+    }
+
+    key := args[0]
+    path := args[1]
+
+    // Retrieve the object from the database
+    obj := store.Get(key)
+    if obj == nil {
+        return []byte("-ERR could not perform this operation on a key that doesn't exist\r\n")
+    }
```

Comment:
in order to keep code style. please use diceerrors.NewErrWithMessage to return err

Reply:
@TheRanomial please address this.